inst/rstudio/addins.dcf

```diff
@@ -3,11 +3,6 @@ Description: Prompt for and set the style used by all STYLER addins
 Binding: prompt_style
 Interactive: true
 
-Name: Style package
```

Comment:
Removing the `Style package` addin is independent from having a selectable style, but I guess the recommended approach going forward will be to call `style_pkg(style = ...)` directly at the console, hence it makes sense to get rid of the addin at this stage.

Reply:
Yes sorry for the inconvenience :-(. In case you want to stick to it, you can add it again manually to the fork after this is merged.

Reply:
Fine with the removal in the context of this fork if you are fine with it :).